### PR TITLE
feat: define new write protocol proto file

### PR DESCRIPTION
**Pull request by xuthus5**
Based on the discussion minutes of the community monthly meeting, a new write protocol proto file definition is now submitted to the SDK.

### CODE REVIEW

File: proto/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# proto
+
+This package contains the OpenGemini protocol buffer definitions.
+
+## Prerequisites
+
+1. Install Protocol Buffers compiler (protoc)
```
Comment (shoothzj) [inline proto/README.md:7]:
```suggestion
1. Install protoc (Protocol Buffers compiler)
```

File: proto/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+2. Install protoc plugins for Go:
+
+```shell
```
Comment (shoothzj) [inline proto/README.md:10]: I guess we can provide a `gen_code.sh`

Comment (xuthus5) [reply proto/README.md:10]: GOOD IDEA.

**Review by shoothzj — COMMENTED** [review]
Let's rm the `proto/write.pb.go` in licenselint ci first.

**Review by shoothzj — APPROVED** [review]

**Pull request merged**